modules/helm-oci-prereqs.adoc

```diff
@@ -8,13 +8,13 @@ $ sudo cp rootCA.pem   /etc/pki/ca-trust/source/anchors/
 $ sudo update-ca-trust extract
 ----
 
-* **Experimental feature:** Many of the commands for interacting with Helm and OCI registries make use of the `helm chart` subcommand.  At the time of writing, OCI support in Helm is still marked as an “experimental” feature and must be enabled explicitly. This is accomplished by setting the environment variable `HELM_EXPERIMENTAL_OCI=1`.
+* **Generally available:** As of Helm 3.8, OCI registry support for charts is now generally available. Many of the commands for interacting with Helm and OCI registries make use of the `helm chart` subcommand.
```

Comment:
In helm 3.8, the chart subcommand is deprecated.

helm chart
Error: unknown command "chart" for "helm"